js/setpassword.js

```diff
@@ -36,6 +36,14 @@
 
 		_resetDone : function(result){
 			if (result && result.status === 'success') {
+				var getRootPath = OC.getRootPath();
+				if (getRootPath === '') {
+					/**
+					 * If owncloud is not run inside subfolder, the getRootPath
+					 * will return empty string
+					 */
+					getRootPath = "/";
+				}
 				OC.redirect(OC.getRootPath());
```

Comment:
@sharidas it is still calling ``OC.getRootPath()`` ddirectly.
And local var ``getRootPath`` looks unusued.

Reply:
Aah copy paste :(